src/ai/backend/test/templates/session/interactive_session.py

```diff
@@ -24,17 +25,32 @@
 )
 
 
-class InteractiveSessionTemplate(WrapperTestTemplate):
-    @property
-    def name(self) -> str:
-        return "interactive_session"
+class _BaseInteractiveSessionTemplate(WrapperTestTemplate):
+    def _build_session_params(
+        self,
+        session_name: str,
+    ) -> dict[str, Any]:
+        image_dep = ImageContext.current()
+        cluster_dep = ClusterContext.current()
+        session_dep = SessionContext.current()
+
+        params: dict[str, Any] = {
+            "image": image_dep.name,
+            "resources": session_dep.resources,
+            "type_": "interactive",
+            "name": session_name,
+            "cluster_mode": cluster_dep.cluster_mode,
+            "cluster_size": cluster_dep.cluster_size,
+        }
+        params.update(self._extra_session_params())
+        return params
```

Comment:
I'm thinking that currently the clientSDK receives arguments directly as parameters, but what if we pass this part as a v2-like form and pass a pydantic object?